modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -812,31 +774,160 @@ values:
 - { name: 'ZIP', type: Integer, length: 5, format: { justify: ZEROES, sign: UNSIGNED } }
 ----
 
-After importing the schema, you can use the `schemaPath` property to reference the flat file through the `output` directive. For example: `output application/flatfile schemaPath="src/main/resources/mailing-record.ffd"`
-
-=== Reader Properties
-
-When defining an input of type Copybook, there are a few optional parameters you can add in the XML definition of your Mule project to customize how the data is parsed.
-
-[%header]
+After importing the copybook, you can use the `schemaPath` property to reference
+the associated flat file through the `output` directive. For example:
+`output application/flatfile schemaPath="src/main/resources/schemas/mailing-record.ffd"`
+
+=== Supported Copybook Features
+
+Not all copybook features are supported by the COBOL Copybook format in
+DataWeave. In general, the format supports most common usages and simple
+patterns, including:
+
+* USAGE of DISPLAY, BINARY (COMP), COMP-5 (_Mule 3 support_: Version 3.9.3 and
+  later), and PACKED-DECIMAL (COMP-3). For character encoding restrictions,
+  see <<character_encoding>>.
+* PICTURE clauses for numeric values consisting only of:
+** '9' - One or more numeric character positions
+** 'S' - One optional sign character position, leading or trailing
+** 'V' - One optional decimal point
+** 'P' - One or more decimal scaling positions
+* PICTURE clauses for alphanumeric values consisting only of 'X' character positions
+* Repetition counts for '9', 'P', and 'X' characters in PICTURE clauses
+  (as in `9(5)` for a 5-digit numeric value)
+* OCCURS DEPENDING ON with controlVal property in schema. Note that if the
+  control value is nested inside a containing structure, you need to manually
+  modify the generated schema to specify the full path for the value in the
+  form "container.value".
+* REDEFINES clause (used to provide different views of the same portion of
```

Comment:
It would be better to have the "see details..." text as a separate sentence

Reply:
fixed added link